client/src/components/pages/ProfileUpdate.tsx

```diff
@@ -86,9 +86,38 @@ const Screen: FC = () => {
   const [profilePath, setProfilePath] = useState('');
   const environment = useRelayEnvironment();
   const envrionmentProps = useRef(environment);
+  const snackbar = useRef<SnackbarRef>(null);
 
   const {user} = useAuthContext();
 
+  const UpdateProfile = useCallback(
+    (type?: SnackbarType): void => {
+      if (snackbar)
+        return snackbar.current?.show({
+          text: getString('UPDATE_PROFILE'),
+          type,
+          styles: {
+            container: {
+              backgroundColor: `${theme.toastBackground}`,
+              width: Dimensions.get('screen').width * 0.9,
+              marginVertical: -(Dimensions.get('screen').height / 12),
```

Comment:
How about using `percentages` here? However, the better one would be `flex`.

If I remember correctly `Dimensions.get('screen').width` do bad calculation when orientation changes. Could you also test this?

Reply:
`Dimensions.get('screen').width`

 As a result of the test, it wasn't good, we expected. 
 So I will change it follow your advice and test it.
 Appreciate you for always giving me advice.